### DISCUSSION TITLE
Add a way to handle projects

### DISCUSSION THREAD

**Discussion by SuperFola** [community/community#5164] [Mobile]
I'm often outside with my mobile, and I would love to be able to handle GitHub projects through the app, so that I won't have to keep track of new ideas on a note app and then copy/paste when I'm back home.

**Comment by Iron-Ham** [comment]
Hey @SuperFola, that's something on our minds too! I'll make sure to add your notes to our internal tracking issue, because I'd love to have this on my phone too.

I'll keep you posted!
